pkg/cmd/env/connect.go

```diff
@@ -0,0 +1,162 @@
+package env
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/coreeng/corectl/pkg/cmdutil/config"
+	"github.com/coreeng/corectl/pkg/cmdutil/userio"
+	corectlenv "github.com/coreeng/corectl/pkg/env"
+	"github.com/coreeng/corectl/pkg/gcp"
+	"github.com/coreeng/developer-platform/pkg/environment"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+)
+
+type EnvConnectOpt struct {
+	Port               int
+	Environment        string
+	RepositoryLocation string
+	ProjectID          string
+	Region             string
+	Streams            userio.IOStreams
+	Exec               corectlenv.Commander
+	gcpClient          gcp.Client
+}
+
+func connectCmd(cfg *config.Config) *cobra.Command {
+	opts := EnvConnectOpt{
+		Exec: corectlenv.NewCommand(),
+	}
+	connectCmd := &cobra.Command{
+		Use:   "connect [environment]",
+		Short: "Connect to an environment",
+		Long:  `This command allows you to connect to a specified environment.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.Streams = userio.NewIOStreams(
+				cmd.InOrStdin(),
+				cmd.OutOrStdout(),
+			)
+			return connect(opts)
+		},
+	}
+
+	connectCmd.Flags().StringVarP(
```

Comment:
This flag represents the same thing as `cfg.Repositories.CPlatform`, which is registered below, doesn't it? If so, let's remove this flag and use cfg.Repositories.CPlatform